src/app/menu/menu.component.html

```diff
@@ -0,0 +1,106 @@
+<p class="title">Main Menu</p>
+<div class="menu">
+    <div class="col-lg-5 col-lg-offset-4">
+        <div class="row">
+            <div class="menu-button">
+                <img class="hex" src="../../assets/img/intro.png">
+                <button class="but intro-btn">
+                    Introduction
+                </button>
+                <div class="intro-outer-circle">
+                    <div class="intro-inner-circle">
+                        <div class="arc first" *ngIf='stage<1 || (stage==1 && activity<1)'></div>
```

Comment:
I think directives would involve too many variables (current stage, current activity, stage and activity of representing divs) so I decided to go ahead with the ngFor implementation.